content/en/docs/tasks/administer-cluster/kubeadm/rotate-certs.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Certificate rotations under Kubeadm
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to rotate the various certificates associated with a kubeadm k8s installation.
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+As part of upgrades, `kubeadm` will check all certificates it manages for expiration.
```

Comment:
Does that means are you planning to change the upgrade process as well? currently, if I remember well it replaces only api server cert older than 180 days

Reply:
Oh is _that_ how it works? I was told is was "3 months from expiry," which I was able to prove wasn't the case.

Reply:
That is my understanding of this code at least ...
https://github.com/kubernetes/kubernetes/blob/0c8fe56ea41d59b9f795949fbced9fde6ab422eb/cmd/kubeadm/app/phases/upgrade/postupgrade.go#L304

Reply:
Agree with @fabriziopandini :)